frontend/dispatcher.lua

```diff
@@ -332,11 +321,40 @@ function Dispatcher:init()
     for i=1,#CreOptions do
         parseoptions(CreOptions, i)
     end
+    UIManager:broadcastEvent(Event:new("DispatcherRegisterActions"))
     Dispatcher.initialized = true
 end
 
+--[[--
+Adds settings at runtime.
+
+@usage
+    function Hello:onDispatcherRegisterActions()
+        Dispatcher:registerAction("helloworld_action", {category="none", event="HelloWorld", title=_("Hello World"), filemanager=true,})
+    end
+
+    function Hello:init()
+        self:onDispatcherRegisterActions()
+    end
+
+
+@param name the key to use in the table
+
+@param value a table per settingsList above
```

Comment:
For me there doesn't need to be a newline in between, but if you prefer that's fine.
```suggestion
@param name the key to use in the table
@param value a table per settingsList above
```